app/src/components/inputs/inputWithIcon/inputWithIcon.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { Input } from 'components/inputs/input';
+import styles from './inputWithIcon.scss';
+
+const cx = classNames.bind(styles);
+
+export const InputWithIcon = ({ icon, iconClass, ...rest }) => (
+  <React.Fragment>
+    <Input {...rest} />
+    <span className={cx('icon', iconClass)}>{icon}</span>
+  </React.Fragment>
+);
+
+InputWithIcon.propTypes = {
+  type: PropTypes.string,
+  value: PropTypes.string,
+  placeholder: PropTypes.string,
+  maxLength: PropTypes.string,
+  disabled: PropTypes.bool,
+  readonly: PropTypes.bool,
+  className: PropTypes.string,
+  icon: PropTypes.string,
+  iconClass: PropTypes.string,
+  error: PropTypes.string,
+  onChange: PropTypes.func,
+  onFocus: PropTypes.func,
+  onBlur: PropTypes.func,
+  onKeyUp: PropTypes.func,
+  onKeyPress: PropTypes.func,
+  refFunction: PropTypes.func,
+  touched: PropTypes.bool,
```

Comment:
it's unnecessary to define propTypes and defaultProps for unused in component props ( rest props in this case )

Reply:
Fixed